redis_cloud/README.md

```diff
@@ -0,0 +1,68 @@
+# Agent Check: Redis Cloud
+
+## Overview
+
+Redis is a versatile and fast data store that supports strings, hashes, lists, sets, streams, and more. It also offers programmability, extensibility, persistence, clustering, and high availability features, as well as Redis Stack for modern data models and processing engines. 
+
+The [Redis Cloud][1] integration is intended for use with Redis CLoud deployments of Redis software. It is not for use with Redis Enterprise installations, for that see the separate Redis Enterprise integration.
+
+This integration provides metrics for three critical aspects of a cluster; database, node, and shard, and makes it easier to detect problems before they become critical. For a full list see the 'Metrics' section.
+
+
+## Setup
+
+### Installation
+
+1. Run the following command to install the Agent integration:
+   ```shell
+   datadog-agent integration install -t datadog-redis_cloud==1.0.0
+   ```
+   
+2. Configure the integration by setting the `openmetrics_endpoint` to your cluster's master node. See [Integration][2] for further information.
+
+3. [Restart][3] the agent.
+
+
+### Configuration
+
+Set the `openmetrics_endpoint` to point to your cluster. See the [example][4].
+
+
+### Validation
+
+1. Ensure you can ping the machine, particularly in a cloud environment. Run `wget --no-check-certificate <endpoint>` or `curl -k <endpoint>` to ensure that you can receive metrics.
+
+2. Check the [status][5] of the Datadog agent.
+
+
+## Data Collected
+
+The current release gathers all metrics for databases, nodes, and shard.
+
```

Comment:
```suggestion
```

Reply:
Are you proposing that we leave the 'Data Collected' section empty, or that we remove it altogether?

Reply:
we've changed this to look like the equivalent section from redis_enterprise, which was already approved